core_backend/add_users_to_db.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
```

Comment:
Is this just for testing?

Reply:
This is for getting users into the user table since in the current setup we don't have user creation endpoints. This just loads the environment variables directly into the user table right when the app initialises.

I don't think it makes sense right now to create complicated user signup/addition workflows - I suspect that we can offload all of that to another service (like [Fief](https://docs.fief.dev/)) and build whatever we need to interface with that user management service when we get there (probably next sprint).